rpcs3/main.cpp

```diff
@@ -508,6 +508,19 @@ int main(int argc, char** argv)
 	setenv( "KDE_DEBUG", "1", 0 );
 #endif
 
+#ifdef __APPLE__
+	struct ::rlimit rlim;
+	::getrlimit(RLIMIT_NOFILE, &rlim);
+	rlim.rlim_cur = OPEN_MAX;
+	if (::setrlimit(RLIMIT_NOFILE, &rlim) != 0)
+		std::cerr << "Failed to set max open file limit (" << OPEN_MAX << ").\n";
```

Comment:
What I meant earlier was that this here should additionally be logged as sys_log.error.
Same goes for the Linux code above

Reply:
Linux uses stderr because the errors aren't supposed to happen on normally configured systems and those who do more obscure stuff (such as disabling overcommit) know to check errors from console if they encounter issues. It should not be used on Mac or Windows as users on these systems will not realize to check stderr.

Reply:
What could possibly be an argument against logging an error to the file here?

Reply:
None, I'm just saying why linux prints to stderr. There was discussion about it earlier.